libraries/botbuilder/src/teamsActivityHandler.ts

```diff
@@ -924,4 +928,108 @@ export class TeamsActivityHandler extends ActivityHandler {
             await handler(teamsChannelData.team, context, next);
         });
     }
+
+    /**
+     * Runs the _event_ sub-type handlers, as appropriate, and then continues the event emission process.
+     *
+     * @param context The context object for the current turn.
+     * @returns A promise that represents the work queued.
+     *
+     * @remarks
+     * Overwrite this method to support channel-specific behavior across multiple channels or to add
```

Comment:
Override*

Reply:
Copy/paste. I'll fix that in all the ActivityHandlers